### PR TITLE
[11.0] partner multi relation tabs

### PR DESCRIPTION
**Pull request by NL66278**

### CODE REVIEW

**Comment by lfreeke** [comment]
:+1: (functional test)

**Comment by NL66278** [comment]
@lfreeke Please use the review button on tab to see changes for review. Otherwise it will not be a valid approval.

**Review by lfreeke — APPROVED** [review]
:+1: (functional test)

File: partner_multi_relation_tabs/models/res_partner_relation_type_selection.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Copyright 2013-2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models
```
Comment (nikul-serpentcs) [inline partner_multi_relation_tabs/models/res_partner_relation_type_selection.py]: odoo

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

File: partner_multi_relation_tabs/models/res_partner_relation_type_selection.py (hunk 1 above)
Comment (NL66278) [reply partner_multi_relation_tabs/models/res_partner_relation_type_selection.py]: @nikul-serpentcs I followed your suggestions.

**Review by ddufresne — APPROVED** [review]
@NL66278 thanks for this module. Very intuitive and works out of the box.

**Pull request merged**